libraries/AP_Notify/ToshibaLED_I2C.cpp

```diff
@@ -29,49 +29,33 @@ extern const AP_HAL::HAL& hal;
 
 bool ToshibaLED_I2C::hw_init()
 {
-    // get pointer to i2c bus semaphore
-    AP_HAL::Semaphore* i2c_sem = hal.i2c->get_semaphore();
-
+    _dev = hal.i2c_mgr->get_device(0, TOSHIBA_LED_ADDRESS);
```

Comment:
You could create a public constructor for this class like you did in some other files to put this `_dev = hal.i2c_mgr->get_device(0, TOSHIBA_LED_ADDRESS);`